manifests/c/CiderCollective/Cider/Nightly/1.6.0-beta.75/CiderCollective.Cider.Nightly.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=QUSU.CRLF.5-1-22621-963.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: CiderCollective.Cider.Nightly
+PackageVersion: 1.6.0-beta.75
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+UpgradeBehavior: uninstallPrevious
+ElevationRequirement: elevationRequired
```

```suggestion
ElevationRequirement: elevationRequired
AppsAndFeaturesEntries:
- DisplayVersion: 1.6.0
  DisplayName: Cider 1.6.0
```
Package is marketed as `1.6.0-beta.75` but version written to ARP/Registry is `1.6.0`.